### PR TITLE
virttest.guest_agent: Add function related to memory block

### PR DESCRIPTION
**Pull request by xiagao**
Add guest-get-memory-block-info,guest-get-memory-blocks and
guest-set-memory-blocks functions.
ID: 1754370
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiangchunfu** [comment]
looks good to me. 
Acked by:Xiangchun Fu xfu@redhat.com

File: virttest/guest_agent.py [hunk 1]
```diff
@@ -875,3 +875,30 @@ def get_osinfo(self):
...
+        Get the list of the guest's memory blocks.
+        """
+        cmd = "guest-get-memory-blocks"
+        self.check_has_command(cmd)
+        return self.cmd(cmd=cmd)
+
+    def set_memory_blocks(self, mem_blocks_list):
+        """
+        Reconfigure (currently: enable/disable) state of guest memory blocks.
+        :param mem_blocks_list: a list of all memory blocks the guest
```
Comment (vivianQizhu) [inline virttest/guest_agent.py]: From my understanding this should not be 'all' memory blocks, but only the ones you want to configure. Isn't it?

Comment (xiagao) [reply virttest/guest_agent.py]: You are right, updated.

**Comment by vivianQizhu** [comment]
Ack.

**Pull request merged**